src/instance/api/events.js

```diff
@@ -137,6 +137,7 @@ export default function (Vue) {
     // then there's no need to broadcast.
     if (!this._eventsCount[event]) return
     var children = this.$children
+    if (typeof children === null) return
```

Comment:
Does the typeof operator ever actually return null? Shouldn't this be `if(children === null)`?